services/src/main/java/org/exoplatform/gamification/connectors/github/rest/HooksManagementRest.java

```diff
@@ -114,4 +114,16 @@ public Response deletehook(@Context UriInfo uriInfo, @PathParam("id") Long id) t
     }
   }
 
+  @GET
+  @RolesAllowed("users")
+  @Path("users/{id}")
+  public Response getUserIdByGithubId(@Context UriInfo uriInfo, @PathParam("id") String githubId) throws Exception {
+    try {
+      String userId = githubHooksManagement.getUserByGithubId(githubId);
+      return Response.ok(userId).build();
```

Comment:
this call may return Null ``` githubHooksManagement.getUserByGithubId(githubId);```
```suggestion
      if(userId != null) {  
        return Response.ok(userId).build();
      } else {
        return Response.status(Response.Status.NOT_FOUND).build();
      }
```

Reply:
Done. I also added empty check.